src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -42,6 +42,9 @@
         package removal but accepts an optional argument for the number of
         days.
     """)
+mask_opts.add_argument(
+    '-b', '--bug', dest='bugs', action='append', type=int,
```

Comment:
Unit tests should be added for both short and long versions of this option (in addition to the non-use case already tested).

See the function at https://github.com/pkgcore/pkgdev/blob/main/tests/scripts/test_pkgdev_mask.py#L184 for an example that can be copied and altered for this case. Note that it should really loop over both short and long opts (-r and --rites) but it looks like I forgot to do that and it only tests the short option currently.

Reply:
ACK on the testing.

Reply:
I implemented tests!